<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientClass</class>
 <widget class="QMainWindow" name="clientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="connexionButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Noto Sans Arabic&quot;;</string>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="speudoLineEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="mot_passeLineEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pseudo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;pseudo :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="connexion">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Connexion&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="motdepasse">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;mot de passe :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="dreamscord">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/reamscord-removebg-preview.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="envoyerButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>400</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>envoyer</string>
    </property>
   </widget>
   <widget class="QLabel" name="incription">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Insciption&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="pseudolabel_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt; Pseudo :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="motDePasselabel_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt; mot de passe :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="confirmationMDPlabel_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;confirmation du mot du passe :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="speudoinscritionlineEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="MDPlineEdit_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmationlineEdit_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="InscriptionButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Inscription</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="messageEnvoyer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>661</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="zone_de_test">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>571</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>759</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="client.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>InscriptionButton</sender>
   <signal>clicked()</signal>
   <receiver>clientClass</receiver>
   <slot>inscription()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connexionButton</sender>
   <signal>clicked()</signal>
   <receiver>clientClass</receiver>
   <slot>connexion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connexionButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>clientClass</receiver>
   <slot>onConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>envoyerButton</sender>
   <signal>clicked()</signal>
   <receiver>clientClass</receiver>
   <slot>message()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>685</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inscription()</slot>
  <slot>connexion()</slot>
  <slot>onConnectButtonClicked()</slot>
  <slot>onInscriptionButtonClicked()</slot>
  <slot>message()</slot>
 </slots>
</ui>
